BepressImportDom.inc.php

```diff
@@ -683,42 +702,64 @@ function _createEmptyAuthor($userGroupId) {
 	 *
 	 * @param $pdfPath string PDF file location path
 	 * @param $locale string|null [Optional] Locale to use. If null, will use primary locale.
+	 * @throws Exception
 	 */
-	function _handlePDFGalleyNode($pdfPath, $locale = null) {
+	private function _handlePDFGalleyNode(string $pdfPath, ?string $locale = null) : void
+	{
 		$pdfFilename = basename($pdfPath);
 
 		// Create a representation of the article (i.e. a galley)
 
+		/** @var ArticleGalleyDAO $articleGalleyDao */
 		$articleGalleyDao = DAORegistry::getDao('ArticleGalleyDAO');
-		$articleGalley = $articleGalleyDao->newDataObject();
-		$articleGalley->setData('publicationId', $this->_submission->getCurrentPublication()->getId());
-		$articleGalley->setName($pdfFilename, $this->_primaryLocale);
-		$articleGalley->setSequence(1);
-		$articleGalley->setLabel('PDF');
-		$articleGalley->setLocale($locale == null ? $this->_primaryLocale : $locale);
-		$articleGalleyDao->insertObject($articleGalley);
+		$newArticleGalley = $articleGalleyDao->newDataObject();
+		$newArticleGalley->setData('publicationId', $this->_submission->getCurrentPublication()->getId());
+		$newArticleGalley->setName($pdfFilename, $this->_primaryLocale);
+		$newArticleGalley->setSequence(1);
+		$newArticleGalley->setLabel('PDF');
+		$newArticleGalley->setLocale($locale == null ? $this->_primaryLocale : $locale);
```

Comment:
Just a shortcut `$locale ?: $this->_primaryLocale`

Reply:
Didn't know that one, thanks!